es5-shim.js

```diff
@@ -1398,23 +1398,25 @@ defineProperties(StringPrototype, {
 // ES5 15.5.4.20
 // whitespace from: http://es5.github.io/#x15.5.4.20
 var ws = '\x09\x0A\x0B\x0C\x0D\x20\xA0\u1680\u180E\u2000\u2001\u2002\u2003' +
-    '\u2004\u2005\u2006\u2007\u2008\u2009\u200A\u202F\u205F\u3000\u2028' +
-    '\u2029\uFEFF';
+	'\u2004\u2005\u2006\u2007\u2008\u2009\u200A\u202F\u205F\u3000\u2028' +
+	'\u2029\uFEFF';
```

Comment:
Also, please rebase this back down to a single commit so there's no "formatting" commits :-)